sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyManagerFactory.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactorySpi;
+import javax.net.ssl.ManagerFactoryParameters;
+import java.security.KeyStore;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * The KeyVault variant of the KeyManagerFactory.
+ */
+public class KeyVaultKeyManagerFactory extends KeyManagerFactorySpi {
+
+    /**
+     * Stores the logger.
+     */
+    private static final Logger LOGGER = Logger.getLogger(KeyVaultKeyManagerFactory.class.getName());
+
+    /**
+     * Stores the key managers.
+     */
+    private final List<KeyManager> keyManagers = new ArrayList<>();
+
+    @Override
+    protected void engineInit(KeyStore keystore, char[] password) {
+        LOGGER.entering(
+            "KeyVaultKeyManagerFactory",
+            "engineInit",
+            new Object[] { keystore, new String(password) }
+        );
+        KeyVaultKeyManager manager = new KeyVaultKeyManager(keystore, password);
+        keyManagers.add(manager);
+    }
+
+    @Override
+    protected void engineInit(ManagerFactoryParameters spec) {
```

Comment:
is this method meant to not do anything ?
Better to throw an UnsupportedOperationException, to let user know,

Reply:
Yes it is meant to do nothing. And the API is defined and it does not state UnsupportedOperationException can be thrown.